### DISCUSSION TITLE
Carrinho de compras

### DISCUSSION THREAD

**Discussion by osvaldoribeirodev** [balta-io/7187#58] [Perguntas e Respostas] [answered]
Acabei de finalizar o curso 7187 e fiquei com uma dúvida.
Como eu faço pra exibir/atualizar ao lado do ícone de carrinho de compras, a quantidade de itens do carrinho?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @osvaldoribeirodev como vai?
Talvez tentar algo como:
`{{ _cart.Length }}`

Me mostra um pedaço de código que consigo te ajudar melhor...
